.github/workflows/build_front.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
```

Comment:
Rename to build&test_front to suit content

Reply:
Sure, will change. It was a first test, I'm not really used to all of this. We'll improve this later !